settings.yaml

```diff
@@ -0,0 +1,123 @@
+HighFrequencyTradingService:
+    Db:
+        LogsConnString:
+            settings-key: LogsConnString
+            types:
+                - AzureBlobStorage
+                - Tag
+        OrderStateConnString:
+            settings-key: HFT-OrderStateConnString
+            types:
+                - MongoDB
+        OrdersArchiveConnString:
+            settings-key: HFT-OrdersArchiveConnString
+            types:
+                - AzureTableStorage
+    
+    MaintenanceMode:
+        Enabled: 
+            settings-key: HFT-MaintenanceMode
+        Reason:
+            settings-key: HFT-MaintenanceReason
+
+    DisabledAssets:
+        settings-key: HFT-DisabledAssets
+        types:
+            - JsonArray
+
+    Dictionaries:
+        CacheExpirationPeriod:
+            settings-key: HFT-DictionaryCacheExpirationPeriod
+
+    Cache:
+        RedisConfiguration:
+            settings-key: HFT-RedisConfiguration
+            types:
+                - Redis
+        ApiKeyCacheInstance:
+            settings-key: HFT-ApiKeyCache
+        ApiKeyCacheKeyPattern:
+            settings-key: HFT-ApiKeyCacheKeyPattern
+        WalletCacheKeyPattern:
+            settings-key: HFT-WalletCacheKeyPattern
+        FinanceDataCacheInstance:
+            settings-key: HFT-FinanceDataCache
```

Comment:
This value must be the same for OrderBooksCacheProvider job. Consider using shared value.